rdagent/components/coder/data_science/share/eval.py

```diff
@@ -52,8 +52,12 @@ def evaluate(
             if self.data_type == "full"
             else self.scen.debug_path
         )
+
+        extra_volumes = {data_source_path: T("scenarios.data_science.share:scen.input_path").r()}
```

Comment:
This duplicated with the mount path.
What if we directly place the  model  under   `<competition>/prev_model/` (i.e. `./workspace_input/prev_model/`) ?